SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -407,6 +407,12 @@ void USpatialWorkerConnection::SetupConnectionConfigFromURL(const FURL& URL, con
 	}
 }
 
+void USpatialWorkerConnection::Tick()
+{
+	QueueLatestOpList();
```

Comment:
This will potentially increase latency on stuff slightly as we queue and send everything at once. Ideally you get ops in tick dispatch and send messages in flush. However for a correctness test this is fine.

Reply:
Addressed.